.devcontainer/dodona-tested.dockerfile

```diff
@@ -0,0 +1,118 @@
+# This is the Dockerfile for the tested judge.
+# It can be downloaded using docker pull dodona/dodona-tested.
+
+# This docker image is run in our production environment.
+# It should not contain any development tools or dependencies.
+# Add those to dev-dependencies.sh instead.
+
+FROM python:3.12.4-slim-bullseye
+
+# Set up the environment
+
+# Kotlin
+ENV SDKMAN_DIR /usr/local/sdkman
+ENV PATH $SDKMAN_DIR/candidates/kotlin/current/bin:$PATH
+ENV PATH $SDKMAN_DIR/candidates/java/current/bin:$PATH
+# Haskell
+ENV HASKELL_DIR /usr/local/ghcupdir
+ENV PATH $HASKELL_DIR/ghc/bin:$PATH
+ENV PATH $HASKELL_DIR/cabal:$PATH
+# Node
+ENV NODE_PATH /usr/lib/node_modules
+
+# Install dependencies
+RUN <<EOF
```

Comment:
This will create only a single layer in the file system? If so, I think going forward our other images can also adopt this style instead of adding everything to a single command. It is much more readable.

Reply:
Yes this only creates a single layer. And I approve of this.